### DISCUSSION TITLE
specify credits separately in AboutDialog?

### DISCUSSION THREAD

**Discussion by ITwrx** [can-lehmann/owlkettle#62] [Q&A] [answered]
Hi, 

The `credits` font in the `AboutDialog` is way too small for my taste. I'm using a stylesheet successfully to increase font-size, but i have to go too big on all the `AboutDialog` text to make the `credits` text large enough. Is there any way to single out the `credits` font in my code and/or stylesheet?

`#AboutDialogue`
```
AboutDialog:
  programName = "Muh App"
  logo = "Muh-App-icon"
  style = [StyleClass("about-dialog")]
  version = "0.9.0"
  credits = @{
    "Created By": @["https://ITwrx.org"],
    "License": @["GPLv3"]
  }
```
`#stylesheet`

`.about-dialog {font-size: 18px;}`

thanks

**Comment by can-lehmann** [comment]
I assume that this is fixed by https://github.com/can-lehmann/owlkettle/commit/bec3124214faccd223defeb44515893d7c72c4e9

**Reply by ITwrx** [reply]
is it possible to specify the `credits` part separately from the rest of the `AboutDialog`? and if so, how?

**Reply by can-lehmann** [reply, marked as answer]
Maybe you could use something like this:

```css
.aboutdialog stack scrolledwindow grid {
	font-size: 30px;
}
```

**Reply by ITwrx** [reply]
thanks! I wasn't sure if the widgets had their own built-in classes or not. I guess i should have opened up the gtk inspector with `ctrl+shift+i` once my program window was in focus. It took a little playing around to figure out how to apply my preferred css to the specific nodes i was targeting, but i arrived at rules which suit my needs well, and it doesn't even require me setting my own class in the nim/owlkettle code.

I ended up with:

```
.aboutdialog box stack {font-size: 22px;}
.aboutdialog box label {font-size: 20px;}
.aboutdialog headerbar label {font-size: 16px;}
.aboutdialog stack scrolledwindow viewport grid {font-size: 22px;}
```
